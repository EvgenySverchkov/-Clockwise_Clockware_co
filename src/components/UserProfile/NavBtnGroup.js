import React from "react";
import {Link} from "react-router-dom";
import LogOutBtn from "./LogOutBtn";

const NavBtnGroup = () => (
<div className="btn-group col-12 p-0 mb-2" role="group">
    <Link type="button" className="btn btn-secondary" to="/client/userProfile/userOrders">Orders</Link>
    <Link type="button" className="btn btn-secondary" to="/client/userProfile/userSettings">Settings</Link>
    <LogOutBtn />
</div>
);

export default NavBtnGroup;