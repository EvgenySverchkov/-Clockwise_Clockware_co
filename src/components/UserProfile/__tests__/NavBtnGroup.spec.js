import React from "react";
import {create} from "react-test-renderer";
import {MemoryRouter} from "react-router-dom";
import {Provider} from "react-redux";
import configMockStore from "redux-mock-store";

import NavBtnGroup from "../NavBtnGroup";

const mockStore = configMockStore();

it("Test of <NavBtnGroup />", ()=>{
    const store = mockStore({
        ordersReducer: {
            currentOrder: {}
        }
    })
    const component = create(
        <MemoryRouter initialEntries={["/"]}>
            <Provider store = {store}>
                <NavBtnGroup/>
            </Provider>
        </MemoryRouter>
    );
    expect(component.toJSON()).toMatchSnapshot();
});