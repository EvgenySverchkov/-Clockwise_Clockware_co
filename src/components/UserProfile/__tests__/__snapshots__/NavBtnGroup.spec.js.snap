// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test of <NavBtnGroup /> 1`] = `
<div
  className="btn-group col-12 p-0 mb-2"
  role="group"
>
  <a
    className="btn btn-secondary"
    href="/client/userProfile/userOrders"
    onClick={[Function]}
    type="button"
  >
    Orders
  </a>
  <a
    className="btn btn-secondary"
    href="/client/userProfile/userSettings"
    onClick={[Function]}
    type="button"
  >
    Settings
  </a>
  <a
    className="btn btn-danger"
    href="/"
    onClick={[Function]}
  >
    Logout
  </a>
</div>
`;
